From a4dbb82074231322d099c9943d7f2929dc711173 Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Fri, 28 Jun 2013 13:20:02 +0100
Subject: [PATCH] License help under CC-BY-SA 3.0 Unported

The help was not explicitly under a license, as described in bug 703238.
Specify the license as CC-BY-SA 3.0 to match with other GNOME help:

https://wiki.gnome.org/DocumentationProject/Licensing

License clarification approved by help contributors (found with
"git shortlog help/C/*.page") as listed below:

Signed-off-by: Jeremy Bicha <jbicha@ubuntu.com>
Signed-off-by: Ekaterina Gerasimova <kittykat3756@gmail.com>
Signed-off-by: Michael Hill <mdhill@gnome.org>
Signed-off-by: Tiffany Ann Antopolski <tiffany.antopolski@gmail.com>
Signed-off-by: Julita Inca <jinca@gnome.org>
---
 help/C/burst-mode.page            | 2 ++
 help/C/effects-apply.page         | 2 ++
 help/C/index.page                 | 2 ++
 help/C/introduction.page          | 2 ++
 help/C/legal.xml                  | 8 ++++++++
 help/C/photo-delete.page          | 2 ++
 help/C/photo-save.page            | 2 ++
 help/C/photo-take.page            | 2 ++
 help/C/photo-view.page            | 2 ++
 help/C/pref-countdown.page        | 2 ++
 help/C/pref-flash.page            | 2 ++
 help/C/pref-fullscreen.page       | 2 ++
 help/C/pref-image-properties.page | 2 ++
 help/C/pref-resolution.page       | 2 ++
 help/C/video-record.page          | 2 ++
 help/Makefile.am                  | 1 +
 21 files changed, 47 insertions(+)
 create mode 100644 help/C/legal.xml

diff --git a/help/C/burst-mode.page b/help/C/burst-mode.page
index c0d64ed..c53593b 100644
--- a/help/C/burst-mode.page
+++ b/help/C/burst-mode.page
@@ -20,6 +20,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc><gui>Burst mode</gui> takes several photos rapidly. Change your pose
     in between each shot!</desc>
   </info>
 
diff --git a/help/C/effects-apply.page b/help/C/effects-apply.page
index 9c36128..73a0014 100644
--- a/help/C/effects-apply.page
+++ b/help/C/effects-apply.page
@@ -15,6 +15,8 @@
       <email>kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Add fun effects to your photos and videos, like turning yourself green
     or distorting the picture like a funhouse mirror.</desc>
   </info>
diff --git a/help/C/index.page b/help/C/index.page
index bfa126c..d2a0a79 100644
--- a/help/C/index.page
+++ b/help/C/index.page
@@ -16,6 +16,8 @@
       <email>kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Cheese lets you take photos and videos using your webcam. You can
     even add special effects!</desc>
 
diff --git a/help/C/introduction.page b/help/C/introduction.page
index da21997..7b5e355 100644
--- a/help/C/introduction.page
+++ b/help/C/introduction.page
@@ -20,6 +20,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Introduction to the <app>Cheese</app> webcam application.</desc>
   </info>
 
diff --git a/help/C/legal.xml b/help/C/legal.xml
new file mode 100644
index 0000000..aa36246
--- /dev/null
+++ b/help/C/legal.xml
@@ -0,0 +1,8 @@
+<license xmlns="http://projectmallard.org/1.0/"
+         href="http://creativecommons.org/licenses/by-sa/3.0/">
+
+  <p>This work is licensed under a
+  <link href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons
+  Attribution-ShareAlike 3.0 Unported License</link>.</p>
+
+</license>
diff --git a/help/C/photo-delete.page b/help/C/photo-delete.page
index 987d5bd..416f167 100644
--- a/help/C/photo-delete.page
+++ b/help/C/photo-delete.page
@@ -16,6 +16,8 @@
       <email>kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Remove photos and videos that you no longer want to keep.</desc>
   </info>
 
diff --git a/help/C/photo-save.page b/help/C/photo-save.page
index c396043..11e83e1 100644
--- a/help/C/photo-save.page
+++ b/help/C/photo-save.page
@@ -16,6 +16,8 @@
       <email>kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Save a photo or video in a different folder.</desc>
   </info>
 
diff --git a/help/C/photo-take.page b/help/C/photo-take.page
index 64be7ec..c3fbcce 100644
--- a/help/C/photo-take.page
+++ b/help/C/photo-take.page
@@ -24,6 +24,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Use your webcam to take photos instead of videos.</desc>
   </info>
 
diff --git a/help/C/photo-view.page b/help/C/photo-view.page
index b0198ca..a133aff 100644
--- a/help/C/photo-view.page
+++ b/help/C/photo-view.page
@@ -16,6 +16,8 @@
       <email>kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Photos and videos will automatically appear in the photo stream.</desc>
   </info>
 
diff --git a/help/C/pref-countdown.page b/help/C/pref-countdown.page
index 976c390..8c0e38a 100644
--- a/help/C/pref-countdown.page
+++ b/help/C/pref-countdown.page
@@ -19,6 +19,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Remove the countdown so that photos are taken instantly.</desc>
   </info>
 
diff --git a/help/C/pref-flash.page b/help/C/pref-flash.page
index 5ec3f8e..5faa86e 100644
--- a/help/C/pref-flash.page
+++ b/help/C/pref-flash.page
@@ -19,6 +19,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Stop the screen from flashing white when taking a photo.</desc>
   </info>
 
diff --git a/help/C/pref-fullscreen.page b/help/C/pref-fullscreen.page
index ca3ad63..c6901be 100644
--- a/help/C/pref-fullscreen.page
+++ b/help/C/pref-fullscreen.page
@@ -10,6 +10,8 @@
       <email>kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Enable and disable fullscreen mode</desc>
   </info>
 
diff --git a/help/C/pref-image-properties.page b/help/C/pref-image-properties.page
index 8f32188..02daa79 100644
--- a/help/C/pref-image-properties.page
+++ b/help/C/pref-image-properties.page
@@ -20,6 +20,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Adjust the saturation, contrast, brightness and hue to make photos and videos look better.</desc>
   </info>
 
diff --git a/help/C/pref-resolution.page b/help/C/pref-resolution.page
index 26bdf87..956c53c 100644
--- a/help/C/pref-resolution.page
+++ b/help/C/pref-resolution.page
@@ -18,6 +18,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>The resolution of a video or photo determines how much detail you can
     see in it, changing it will affect the file size.</desc>
 
diff --git a/help/C/video-record.page b/help/C/video-record.page
index 315304a..f4b01bd 100644
--- a/help/C/video-record.page
+++ b/help/C/video-record.page
@@ -19,6 +19,8 @@
       <email its:translate="no">kittykat3756@gmail.com</email>
     </credit>
 
+    <include href="legal.xml" xmlns="http://www.w3.org/2001/XInclude"/>
+
     <desc>Use your webcam to record short movies for sharing with friends.</desc>
   </info>
 
diff --git a/help/Makefile.am b/help/Makefile.am
index 60d2703..76529d6 100644
--- a/help/Makefile.am
+++ b/help/Makefile.am
@@ -19,6 +19,7 @@ HELP_FILES = \
 	effects-apply.page \
 	index.page \
 	introduction.page \
+	legal.xml \
 	photo-delete.page \
 	photo-save.page \
 	photo-take.page \
